test/test_format.py

```diff
@@ -12,8 +12,10 @@ def test_formatting():
 
 
 def check_format(implementation: pqclean.Implementation):
-    cfiles = glob(os.path.join(implementation.path(), '*.c'))
-    hfiles = glob(os.path.join(implementation.path(), '*.h'))
+    if shutil.which('astyle') is None:
+        raise unittest.SkipTest("AStyle is not installed")
```

Comment:
Do we want to skip if astyle is not found or report a failure?  If our CI systems suddenly don't have astyle, we would want to know these are failing, not being skipped.

Reply:
I guess I could add some detection for the CI environment, I did not want to push annoying failures on people in places where it's hard to install these tools (windows)

Reply:
See https://github.com/PQClean/PQClean/pull/70/commits/b2c8a136607ddd7c696c185c47b7b016f8548a01